packages/host/app/components/ai-assistant/past-sessions.gts

```diff
@@ -1,182 +1,65 @@
-import { fn, array, hash } from '@ember/helper';
 import { on } from '@ember/modifier';
 import Component from '@glimmer/component';
 
-import { format as formatDate } from 'date-fns';
-import ToElsewhere from 'ember-elsewhere/components/to-elsewhere';
-
-import {
-  BoxelDropdown,
-  IconButton,
-  Menu,
-  Tooltip,
-} from '@cardstack/boxel-ui/components';
-import { eq, menuItem } from '@cardstack/boxel-ui/helpers';
-import {
-  Upload,
-  IconPencil,
-  IconTrash,
-  ThreeDotsHorizontal,
-} from '@cardstack/boxel-ui/icons';
-
-import type { AiSessionRoom } from '@cardstack/host/components/ai-assistant/panel';
-import DeleteModal from '@cardstack/host/components/operator-mode/delete-modal';
+import { IconButton } from '@cardstack/boxel-ui/components';
+import { eq } from '@cardstack/boxel-ui/helpers';
+import { DropdownArrowFilled } from '@cardstack/boxel-ui/icons';
 
 import type { RoomField } from 'https://cardstack.com/base/room';
 
+import AiAssistantPanelPopover from './panel-popover';
+import PastSessionItem, { type RoomActions } from './past-session-item';
+
 interface Signature {
   Args: {
-    sessions: AiSessionRoom[];
-    openSession: (roomId: string) => void;
-    renameSession: (room: RoomField) => void;
-    deleteSession: (roomId: string) => void;
-    roomToDelete: RoomField | undefined;
-    setRoomToDelete: (room: RoomField | undefined) => void;
+    sessions: RoomField[];
+    roomActions: RoomActions;
+    onClose: () => void;
   };
-  Element: HTMLButtonElement;
+  Element: HTMLElement;
 }
 
 export default class AiAssistantPastSessionsList extends Component<Signature> {
   <template>
+    <AiAssistantPanelPopover data-test-past-sessions ...attributes>
+      <:header>
+        Past Sessions
+        <IconButton
+          @icon={{DropdownArrowFilled}}
+          @width='12px'
+          @height='12px'
+          {{on 'click' @onClose}}
+          aria-label='Close Past Sessions'
+          data-test-close-past-sessions
+        />
+      </:header>
+      <:body>
+        {{#if (eq @sessions.length 0)}}
```

Comment:
I see that this is just moved from below, I’m noticing now it could be slightly simplified like this:

```
{{#if @sessions}}
… (list sessions)
{{else}}
… No past sessions
{{/if}}
```